### ISSUE TITLE
[Accessibility] Inputs.Json: Keyboard outline focus is lost while navigating to 'FlightDetails.Json' using Up/Down key.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6273]
### Target Platforms

UWP

### SDK Version

Application Version: 2.7.2108.2001

### Application Name

Visualizer

### Problem Description

[35353187](https://microsoft.visualstudio.com/OS/_workitems/edit/35353187)

Keyboard focus should not be lost while navigating to 'FlightDetails.Json'. Elements after 'FlightDetails.Json' should be navigable after pressing the down arrow key.

### Card JSON

[FlightDetails.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.2/Scenarios/FlightDetails.json)

**Comment by jwoo-msft** [comment]
voice over focus is lost when the card displays errors from app.
there is no reason to think this card behaves differently.
fixing the error messages should be sufficient to close this issue.

**Comment by carlos-zamora** [comment]
This issue is filed on the UWP visualizer, which does not ship to the customer. Additionally, this issue does not repro.
